lib/confirmation.rb

```diff
@@ -0,0 +1,22 @@
+require 'twilio-ruby'
+
+class Confirmation
+
+  def send_text
+    account_sid = 'AC4bcca295ad97a98cacde7dc983f7d14a'
```

Comment:
Strongly advise removing this data from GitHub, and using environment variables instead to store it locally and access it when running the code.